### PR TITLE
chore(hero-cover): refactor hero-cover

### PR DESCRIPTION
**Pull request by zwaardje**

### CODE REVIEW

File: src/js/components/organisms/hero-cover/hero-cover.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+.hero--cover {
+    @extend %reset-last-margin;
```
Comment (matijs) [inline src/js/components/organisms/hero-cover/hero-cover.scss]: This needs to go.

File: src/js/components/organisms/hero-cover/hero-cover.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        color: $colorWhite;
+    }
+}
+
+.hero-cover__headline {
+    color: $colorWhite;
+}
+
+.hero-cover__image--full {
+    @include object-fit(cover);
```
Comment (matijs) [inline src/js/components/organisms/hero-cover/hero-cover.scss]: This is a `<div>`. `object-fit` only works on replaced media such as ``. Also, the mixin is abstracting away useful information.

File: src/js/components/organisms/hero-cover/hero-cover.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ import WithRender from './hero-cover.vue.html';
     name: 'hero-cover',
 })
 
-export default class HeroCover extends Vue implements IHeroCover {
+export class HeroCover extends Vue implements IHeroCover {
```
Comment (matijs) [inline src/js/components/organisms/hero-cover/hero-cover.ts]: Should `IHeroCover` be `HeroCoverProps` to be more consistent?

File: src/scss/atomic/organisms/_all.scss [hunk 4]
```diff
@@ -30,3 +29,4 @@
 @import "~organisms/search-results/search-results";
 @import "~organisms/tabbed-table/tabbed-table";
 @import "~organisms/table-compare/table-compare";
+@import "~organisms/hero-cover/hero-cover";
```
Comment (matijs) [inline src/scss/atomic/organisms/_all.scss]: Can you order the organisms alphabetically?

File: src/js/components/organisms/hero-cover/hero-cover.scss (hunk 1 above)
Comment (zwaardje) [reply src/js/components/organisms/hero-cover/hero-cover.scss]: Not going into css yet. Only moving components for now. Css is the next stage / stage.

File: src/js/components/organisms/hero-cover/hero-cover.scss (hunk 2 above)
Comment (zwaardje) [reply src/js/components/organisms/hero-cover/hero-cover.scss]: Not going into css yet. Only moving components for now. Css is the next stage / stage.

**Pull request merged**